sdk/redis/Microsoft.Azure.Management.RedisCache/src/Microsoft.Azure.Management.Redis.csproj

```diff
@@ -6,10 +6,11 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Redis</PackageId>
     <Description>Provides redis cache management capabilities for Microsoft Azure.</Description>
-    <Version>4.4.1</Version>
+    <Version>4.4.2</Version>
```

Comment:
Since you are adding new features, please consider updating the minor version instead of the patch number.
If you (or the new api version) are making a breaking change, update the major version.

Please also Update the following information in AssemblyInfo.cs
- AssemblyFileVersion 
- AssemblyVersion (If it's a major version update)

Reply:
@isra-fel 
Updated the major version as there is new param MinimumTlsVersion added to Create, Get and Update operations